lib/DDG/Spice/Currency.pm

```diff
@@ -153,7 +153,10 @@ handle query_lc => sub {
         }
 
         # if only a currency symbol is present without "currency" keyword, then bail.
-	return if ($amount eq '' && $to eq '' && $currencyKeyword eq '' && exists($currHash{$from}));
+        return if ($amount eq '' && $to eq '' && $currencyKeyword eq '' && exists($currHash{$from}));
+        
+        # for edge cases that we don't want to trigger on
+        return if $from eq 'mop' && $to eq 'top';
```

Comment:
We need to bail when the query is just `mop tops` --- this prevents queries like `1 mop to top`